main.go

```diff
@@ -18,18 +18,19 @@ func main() {
 	setup()
```

Comment:
Optimization is require on this function as it carries more functions(risky), in its core.